src/main/java/domain/Line.java

```diff
@@ -0,0 +1,34 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Line {
+    private final List<Boolean> points;
+
+    public Line(int columnCount, ConnectionStrategy strategy) {
+        this.points = generateLine(columnCount, strategy);
+    }
+
+    public List<Boolean> getPoints() {
+        return points;
+    }
+
+    private List<Boolean> generateLine(int columnCount, ConnectionStrategy strategy) {
```

Comment:
```suggestion
    public Line(List<Boolean> points) {
        // TODO: 유효성 검증
        this.points = points;
    }

    public Line(int columnCount, ConnectionStrategy strategy) {
        this(generateLine(columnCount, strategy));
    }
    
    public List<Boolean> getPoints() {
        return points;
    }

    private static List<Boolean> generateLine(int columnCount, ConnectionStrategy strategy) {
```

주 생성자를 두고 라인을 만들 수 있게 만들어봐도 되지 않았을까요?
그러면 라인을 쉽게 만들 수 있는 구조가 될 것 같아요